### PR TITLE
Modify README file

### PR DESCRIPTION
**Pull request by Mikachu2333**
1. Renformat Readme
2. Add an example
3. Clearly introduct range of each opinion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +85,16 @@ List of available resize filters:
 - `mitchell` => Resize using Mitchell-Netravali filter
 - `lanczos3` => Resize using Sinc-windowed Sinc with radius of 3
 
+## Example
```
Comment (SalOne22) [inline README.md:88]: If not hard, please add examples for image resizing and quantization

**Review by SalOne22 — CHANGES_REQUESTED** [review]
Overall good, but section that auto generated by clap needs improvements. I also add this changes to source code, to match readme after this PR.

File: README.md [hunk 2]
```diff
@@ -42,28 +42,35 @@ Options:
...
 
 General:
-  -q, --quality <QUALITY>  Optimization quality [default: 75]
-  -f, --codec <CODEC>      Image codec to use [default: mozjpeg]
-  -o, --output <DIR>       Write output file(s) to <DIR>
-  -r, --recursive          Saves output file(s) preserving folder structure
-  -s, --suffix [<SUFFIX>]  Appends suffix to output file(s) names
-  -b, --backup             Appends '.backup' to input file(s) names
+  -q, --quality <QUALITY>         Optimization quality
+                                  [default: 75, Range: 1-100]
```
[the remark this answers is not in the document]
Comment (Mikachu2333) [reply README.md:46]: ok, I'll correct it

File: README.md [hunk 3]
```diff
@@ -42,28 +42,35 @@ Options:
...
+      --dithering [<QUALITY>]     Enables dithering with optional quality
+                                  [default: 75, Range: 1-100]
 
 Resizing:
-      --width <WIDTH>    Resize image with specified width
-      --height <HEIGHT>  Resize image with specified height
-      --filter <FILTER>  Filter used for image resizing [default: lanczos3]
+      --width <WIDTH>             Resize image with specified width
+                                  [Integer only]
```
[the remark this answers is not in the document]
Comment (Mikachu2333) [reply README.md:62]: no problem

File: README.md (hunk 1 above)
Comment (Mikachu2333) [reply README.md:88]: ok, I'll add another example for that.

**Review by SalOne22 — APPROVED** [review]
All looks good

**Pull request merged**